content/id/blogs/working-with-multiple-gitconfig.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Bekerja Menggunakan Lebih dari Satu Gitconfig"
+date: 2023-06-19T20:51:59+07:00
+draft: false
+iscjklanguage: false
+isarchived: false
+categories: ["tech"]
+images: ["https://images.unsplash.com/photo-1647166545674-ce28ce93bdca?w=1920&q=50"]
+aliases: []
+description: "pakai banyak gitconfig, pisahkan persona git kalian"
+---
+#### Translated by: [@paramithatm](https://github.com/paramithatm)
+
+{{< unsplash user="@praveentcom" src="photo-1647166545674-ce28ce93bdca" q="50" >}}
+
+## Pengenalan
+
+Ini adalah tips singkat untuk menggunakan beberapa git config pada 1 mesin lokal. Jika kalian bekerja menggunakan Gitlab/Bitbucket/Github dengan beberapa email/username/gpgsign yang berbeda, atau jika kalian menggunakan akun personal dan akun kerja kalian di mesin yang sama.
+
+## Mengatur Direktori Gitconfig
+
+Nama direktori bebas, tapi demi kemudahan penamaan, mari kita namai `~/.gitconfig.d/`. 
+
+```shell
+$ mkdir -p "$HOME/.gitconfig.d/" && cd "$_"
+$ pwd
+/Users/clavianus.juneardo/.gitconfig.d
+```
+
+## Mengatur Masing-masing Gitconfig
+
+Setelah membuat direktori, misalnya kalian mau mengatur akun `personal` dan `kerja`:
+
+### Mengatur Gitconfig Personal
+
+```shell
+$ git config --file=personal user.name "foo"
+$ git config --file=personal user.email "foo@gmail.com"
+$ git config --file=personal core.editor "vim"
+...
+```
+
+Dengan menggunakan penanda `--file=personal`, gitconfig tersebut akan dikonfigurasi di dalam _file_ `personal`.
```

Comment:
*flag* is technical term, I don't think we need to translate it


```suggestion
Dengan menggunakan flag `--file=personal`, gitconfig tersebut akan dikonfigurasi di dalam _file_ `personal`.
```

Reply:
done